src/util/std_code.h

```diff
@@ -143,7 +143,7 @@ class code_blockt:public codet
       }
       else if(statement==ID_label)
       {
-        assert(last->operands().size()==1);
+        DATA_INVARIANT(last->operands().size()==1, "label has one operand");
```

Comment:
Phrase as a "must" or "should" statement, otherwise you get a confusing crash dump like "Invariant failed: label has one operand" which suggests the label having one operand is wrong.